adminStuff/addNewPost.php

```diff
@@ -0,0 +1,35 @@
+<?php
+include('config/init.php');
+if(isset($_REQUEST['submitItem'])){
+    $errors = array();
+
+    if(!@$_REQUEST['blogTitle']){
+        $errors['blogTitle'] = "*Required*";
+    }
+
+    if(!@$_REQUEST['body']){
+        $errors['body'] = "*Required*";
+    }
+    if(!@$_REQUEST['tagName']){
+        $errors['tagName'] = "*Required*";
+    }
+    if(sizeof($errors) == 0){
+        echo addPostQuery($_REQUEST['blogTitle'], $_REQUEST['body'], $_REQUEST['tagName']);
+        exit;
+    }
+}
+
+echo echoAdminHeaderHtml("Admin Section", "Edit Post").
+        "</div><h2> New Post </h2><div class = 'forms'></br>
+            <form action='' method='post'>
+            <input type='hidden' name='dateOfPost' id='dateOfPost' value=".getPostDateTime(). " />".
```

Comment:
You can do this, but it might be easier to just look up the current date/time in the function that does the insert query. That way you don't need this hidden input at all. It's also technically a bit more accurate because it'll give you the time the post was submitted rather than the time the html for the form was loaded initially.